package.json

```diff
@@ -12,33 +12,30 @@
     "@rescui/typography": "0.0.1",
     "@rescui/ui-contexts": "0.0.2",
     "@rollup/plugin-buble": "0.21.3",
+    "algoliasearch": "4.10.2",
     "aos": "2.1.1",
     "autoprefixer": "6.3.1",
     "babel-core": "6.18.2",
     "babel-loader": "7",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "clean-webpack-plugin": "0.1.18",
     "codemirror": "5.61.0",
     "core-js": "^2.4.1",
-    "css-loader": "2.1.0",
+    "css-loader": "5.2.6",
     "csso-webpack-plugin": "2.0.0-beta.1",
     "debounce": "1.0.2",
     "event-emitter": "0.3.4",
-    "exports-loader": "0.7.0",
     "extend": "3.0.2",
-    "file-loader": "3.0.1",
+    "file-loader": "6.2.0",
     "fixer.js": "1.4.15",
     "highlight.js": "10.5.0",
-    "imports-loader": "0.8.0",
     "in-viewport": "3.4.2",
-    "instantsearch.js": "2.0.2",
+    "instantsearch.js": "4.24.1",
     "jquery": "3.5.0",
     "kotlin-playground": "1.22.0",
-    "mini-css-extract-plugin": "0.5.0",
+    "mini-css-extract-plugin": "1.6.1",
     "mini-svg-data-uri": "1.1.3",
-    "monkberry-loader": "4.0.9",
```

Comment:
I haven’t found any ".monk" template and decided to remove the "monkberry-loader"